lib/fs.js

```diff
@@ -550,9 +550,23 @@ function read(fd, buffer, offset, length, position, callback) {
 
   validateOffsetLengthRead(offset, length, buffer.byteLength);
 
-  if (!NumberIsSafeInteger(position))
+  if (position == null)
     position = -1;
```

Comment:
Not necessarily something to be done in this PR and I haven't looked super-closely but I wonder if this bit of code means that they docs shouldn't say the default is `null` but instead should say the default is `-1` (and be sure to explain what `-1` means and make sure that passing `-1` explicitly works). Saying the thing has to be a an integer (or bigint, as added in this PR), but then saying the default is something that is not an integer or bigint (`null`), seems confusing and not quite right.

Reply:
I agree. I was also thinking about removing the `null`-check and using a `default function parameters` for `position` by setting it to `-1` in the function prototype itself. Should I add that change?